docs/resources/aws_iam_write.md

```diff
@@ -0,0 +1,194 @@
+  depends_on = [p0_aws_staged.example]
+  login = {
+    type = "iam"
+    identity = {
+      type = "email"
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The AWS account ID
+- `login` (Attributes) How users log in to this AWS account (see [below for nested schema](#nestedatt--login))
+
+### Optional
+
+- `label` (String) The AWS account's alias (if available)
+
+### Read-Only
+
+- `state` (String) This account's install progress in the P0 application:
+			- 'stage': The account has been staged for installation
+			- 'configure': The account is available to be added to P0, and may be configured
+			- 'installed': The account is fully installed
+
+<a id="nestedatt--login"></a>
+### Nested Schema for `login`
+
+Required:
+
+- `type` (String) One of:
+    - 'iam': Users log in as IAM users; 'identity' attribute is required
+    - 'idc': Users log in via Identity Center (formerly 'SSO'); 'parent' attribute is required
+    - 'federated': Users log in via a federated identity provider; 'provider' attribute is required
+
+Optional:
+
+- `identity` (Attributes) How user identities are mapped to AWS IAM users (see [below for nested schema](#nestedatt--login--identity))
+- `parent` (String) Identity Center parent account ID
+- `provider` (Attributes) Federated login provider details (see [below for nested schema](#nestedatt--login--provider))
+
+<a id="nestedatt--login--identity"></a>
+### Nested Schema for `login.identity`
+
+Required:
+
+- `type` (String) One of:
+    - 'email': IAM user names are user email addresses
+    - 'tag': User email addresses appear in IAM user tag; 'tag_name' is required
+
+Optional:
+
+- `tag_name` (String) The name of the AWS user tag that holds the user's email address
+
+
+<a id="nestedatt--login--provider"></a>
+### Nested Schema for `login.provider`
+
+Required:
+
+- `app_id` (String) Okta AWS federation app ID
+- `identity_provider` (String) AWS provider integration; this is the _name_ of the AWS integration that you use for federated login,
+defined on the ["Identity providers" tab](https://console.aws.amazon.com/iam/home#/identity_providers) of your IAM dashboard
+- `method` (Attributes) The federation method used by your identity provider (see [below for nested schema](#nestedatt--login--provider--method))
+
+Optional:
+
+- `type` (String) Only 'okta' is supported at this time
+
+<a id="nestedatt--login--provider--method"></a>
+### Nested Schema for `login.provider.method`
+
+Required:
+
+- `account_count` (Attributes) Number of AWS accounts linked to the federation app:
+    - 'single': One account only
+    - 'multi': Multiple accounts, via a parent account (see [below for nested schema](#nestedatt--login--provider--method--account_count))
+
+Optional:
+
+- `type` (String) Only 'saml' is supported at this time
+
+<a id="nestedatt--login--provider--method--account_count"></a>
+### Nested Schema for `login.provider.method.type`
```

Comment:
This appears to be a bug in the doc generator; it should be `login.provider.method.account_count`. 🤷